.atlascli.goreleaser.yml

```diff
@@ -51,7 +51,7 @@ archives:
   wrap_in_directory: false
   format: zip
 nfpms:
-- package_name: mongodb-atlas
+- package_name: mongodb-atlas-cli
```

Comment:
are you sure, last conversation I saw `mongodb-atlas` was the name for this

Reply:
Yes, they have changed it this morning https://docs.google.com/document/d/1vV7S0t4DHwDLFfr2zb1C28mAAhmjP-KcVzmr0gotoF4/edit?disco=AAAAVbOFGBc

Reply:
yes, they have changed it this morning in the PD